index.qmd

```diff
@@ -210,3 +210,7 @@ shiny create --template dashboard-tips
 
 
 :::::
+
+<p class="h4">
+[Get Started](get-started/index.qmd) with building your first Shiny Application now!
+</p>
```

Comment:
this section didn't seem to update in the preview

Reply:
It's there now but it seems to have broken the page a bit. Also I think this should go inside the blue area